### PR TITLE
COMMERCE-11410 Add a new functional test

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-11410

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 1]
```diff
@@ -914,6 +914,53 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11386. As admin, I can cancel an order without getting an error in the log."
+	@priority = 5
+	test CanCancelAnOrder {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:919]:
```suggestion
	test CanCancelOrder {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**